lightning/src/ln/channelmanager.rs

```diff
@@ -924,6 +925,15 @@ pub(crate) const MPP_TIMEOUT_TICKS: u8 = 3;
 /// [`OutboundPayments::remove_stale_resolved_payments`].
 pub(crate) const IDEMPOTENCY_TIMEOUT_TICKS: u8 = 7;
 
+/// The maximum number of unfunded channels we can have per-peer before we start rejecting new
+/// (inbound) ones. The number of peers without funded channels is limited separately in
+/// [`MAX_NO_CHANNEL_PEERS`].
+const MAX_UNFUNDED_CHANS_PER_PEER: usize = 4;
+
+/// The maximum number of peers which we do not have a (funded) channel with. Once we reach this
+/// many peers we reject new (inbound) connections.
+const MAX_NO_CHANNEL_PEERS: usize = 50;
```

Comment:
This constant seems to be used in 2 different ways:

- the max number of connected peers with only unfunded channels OR with no channels
- the max number of peers with only unfunded channels

This is confusing to me.  Perhaps there should be two constants?